google/cloud/forseti/services/inventory/storage.py

```diff
@@ -158,11 +155,35 @@ def set_error(self, session, message):
             session (object): session object to work on.
             message (str): Error message to set.
         """
-
         self.inventory_index_errors = message
         session.add(self)
         session.flush()
 
+    def get_summary(self, session):
+        """Generate/return an inventory summary for this inventory index.
+
+        Args:
+            session (object): session object to work on.
+
+        Returns:
+            dict: a (resource type -> count) dictionary
+        """
+        column = Inventory.resource_type
+        return dict(
+            session.query(column, func.count(column))
```

Comment:
Please change `column` to `resource_type`.... it will be more easy to see what the code is counting and grouping.

Reply:
Done.